src/util/ChartUtils.ts

```diff
@@ -759,7 +758,6 @@ export const offsetPositive = (series: any) => {
       } else {
         series[i][j][0] = 0;
         series[i][j][1] = 0;
-        negative = series[i][j][1];
```

Comment:
Also not used. Probably copied from a previous version like `offsetNegative` or somewhere else.